src/background/intial-assessment-store-data-generator.ts

```diff
@@ -14,6 +14,7 @@ import {
     PersistedTabInfo,
     RequirementIdToResultMap,
 } from '../common/types/store-data/iassessment-result-data';
+import { IDictionaryStringTo } from '../scanner/dictionary-types';
```

Comment:
>IDictionaryStringTo [](start = 9, length = 19)

(per offline discussion) Can we be consistent about `DictionaryStringTo` vs `IDictionaryStringTo`?

Reply:
Done.